alliance/apps/backlog/views/list_backlogs_view.py

```diff
@@ -30,13 +38,9 @@ def get(self, request):
         for backlog in backlogs:
             read_only = backlog.status.id == queued_status_id()
 
-            try:
-                estimate = Estimate.objects.get(team_id=team_id,
-                                                backlog_id=backlog.id)
-            except Estimate.DoesNotExist:
-                estimate = Estimate()
-                estimate.team_id = team_id
-                estimate.backlog_id = backlog.id
+            estimate = build_estimate(Estimate.objects.get_object_or_none(
```

Comment:
I don't think you want `Estimate.objects.get_object_or_none(...)`, you want `get_object_or_none(Estimate, ...)`